docs/install_guides/pypi-openvino-dev.md

```diff
@@ -19,7 +19,7 @@ OpenVINO™ toolkit is a comprehensive toolkit for quickly developing applicatio
-| [Model Downloader and other OMZ tools](https://docs.openvinotoolkit.org/latest/omz_tools_downloader.html)| `omz_downloader` <br> `omz_converter` <br> `omz_quantizer` <br> `omz_info_dumper`| **Model Downloader** is a tool for getting access to collection of high quality and extremely fast pre-trained deep learning [public](https://docs.openvinotoolkit.org/latest/omz_models_group_public.html) and [intel](https://docs.openvinotoolkit.org/latest/omz_models_group_intel.html)-trained models. These free pre-trained models can be used instead of training your own models to speed-up the development and production deployment process.  The principle of the tool is as follows: it downloads model files from online sources and, if necessary, patch them to make them more usable with Model Optimizer. A number of additional tools are also provided to automate the process of working with downloaded models:<br> **Model Converter** is a tool for conversion the models that are stored not in the Inference Engine IR format into that format using Model Optimizer. <br> **Model Quantizer** is a tool for automatic quantization full-precision models in the IR format into low-precision versions using Post-Training Optimization Tool. <br> **Model Information Dumper** is helper utility for dumping information about the models in a stable machine-readable format.|
+| [Model Downloader and other OMZ tools](https://docs.openvinotoolkit.org/latest/omz_tools_downloader.html)| `omz_downloader` <br> `omz_converter` <br> `omz_quantizer` <br> `omz_info_dumper`| **Model Downloader** is a tool for getting access to collection of high quality and extremely fast pre-trained deep learning [public](https://docs.openvinotoolkit.org/latest/omz_models_group_public.html) and [intel](https://docs.openvinotoolkit.org/latest/omz_models_group_intel.html)-trained models. These free pre-trained models can be used instead of training your own models to speed-up the development and production deployment process.  The principle of the tool is as follows: it downloads model files from online sources and, if necessary, patches them to make them more usable with Model Optimizer. A number of additional tools are also provided to automate the process of working with downloaded models:<br> **Model Converter** is a tool for converting the models that are stored not in the Inference Engine IR format into that format using Model Optimizer. <br> **Model Quantizer** is a tool for automatic quantization full-precision models in the IR format into low-precision versions using Post-Training Optimization Tool. <br> **Model Information Dumper** is a helper utility for dumping information about the models in a stable machine-readable format.|
 
```

Comment:
Please say "Open Model Zoo tools" instead of "OMZ tools" for those seeing the acronym the first time